src/components/Dashboard/DirectorListSm.vue

```diff
@@ -85,6 +92,14 @@ export default class DirectorListSm extends Mixins(CountriesProvincesMixin, Comm
 // Variables
 $avatar-width: 2.75rem;
 
+// Complete filing required styling
+.complete-filing {
+  padding: 2rem;
```

Comment:
You don't need padding here and in the div's class, do you?

Reply:
Nope! Actually re-tested this and found my styling was incorrect, so fixed that.
I kept the vertical padding to the div as it wasn't quite working on the span.